src/pages/summer/Main/Feed/Comments/Comments.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import './Comments.scss';
+
+class Comments extends React.Component {
+  render() {
+    const { comments, likes, handleClickDel, handleClickLike } = this.props;
+
+    return (
+      <ul className="detail-comments">
+        {comments.map((cmt, i) => (
+          <li key={cmt.commentId} className="comment">
+            <em className="user-name">{cmt.userName}</em>
+            <span>{cmt.content}</span>
+            <div className="comment-icons">
+              <i
+                index={i}
+                onClick={() => handleClickLike(cmt.commentId)}
+                className={
+                  likes[i] ? 'fas fa-heart cmt-like' : 'far fa-heart cmt-like'
+                }
```

Comment:
```suggestion
                className={`fa-heart cmt-like ${likes[i] ? "fas" : "far"}`}
```
변화하는 속성에 대해서만 삼항 연산자로 처리해주어도 될거 같습니다